base_layer/wallet/src/output_manager_service/recovery/standard_outputs_recoverer.rs

```diff
@@ -151,6 +152,7 @@ where
                 &self.rewind_data,
                 None,
                 Some(proof),
+                OutputSource::Recovered,
```

Comment:
Even when recovered, the output should be marked according to its script (one-sided, stealth, standard, etc.)